src/Engines/MeilisearchEngine.php

```diff
@@ -138,13 +139,6 @@ protected function performSearch(Builder $builder, array $searchParams = [])
     {
         $meilisearch = $this->meilisearch->index($builder->index ?: $builder->model->searchableAs());
 
-        // meilisearch-php 0.19.0 is compatible with meilisearch server 0.21.0...
-        if (version_compare(MeiliSearch::VERSION, '0.19.0') >= 0 && isset($searchParams['filters'])) {
-            $searchParams['filter'] = $searchParams['filters'];
-
-            unset($searchParams['filters']);
-        }
-
         if ($builder->callback) {
             $result = call_user_func(
                 $builder->callback,
```

Comment:
@brunoocasali I dont remember the exact logic behind using rawSearch but as far as I can tell, all methods expect a plain PHP array and therefore this was the way.

furthermore users can modify the query and apply custom logic https://laravel.com/docs/9.x/scout#customizing-engine-searches if a user returns a `SearchResult` object by returning the result of `$meilisearchClient->search()` we read the `getRaw()` of the `SearchResult`object

additionally the `rawSearch` method has less overhead. If there is value in switching to `->search()` instead of `->rawSearch()` please tell me. for v1, I suppose there is no change in the result object so we can safely stick to the plain array.

the implementation tries to stick as close to the aloglia driver as possible which also returns a plain php array.

Reply:
The search methods will stay as they were during the v1 transition.

I'm taking this opportunity to clean up the repository. I'll list every possible breaking-change in the next changelog, but as far as I checked, nothing will impact the work of this branch.

Reply:
@brunoocasali as far as I can tell from the open PR https://github.com/meilisearch/meilisearch-php/pull/454 only the `deleteAllIndexes` impacts the code in the console commands of this repo.

I subscribed to your PR so I can update the code here as soon as that one was merged